backend/prompt_studio/prompt_studio_core_v2/views.py

```diff
@@ -380,6 +381,13 @@ def create_profile_manager(self, request: HttpRequest, pk: Any = None) -> Respon
             raise MaxProfilesReachedError()
         try:
             self.perform_create(serializer)
+            # Check if this is the first profile and make it default for all prompts
```

Comment:
NIT: Ideally this check for max profile count should have been in the serializer since its a validation being done. Similar this logic of updating another entity could have been done in the serializer's `create()` to allow for this logic to kick in at all places where the `ProfileManagerSerializer` is used